### ISSUE TITLE
Code

### ISSUE THREAD

**Issue by qihua28** [iGuoYanjun/C2ANet#3]
感谢作者杰出的工作，我想咨询一下关于代码当中 depth[0]*2]为什么要乘以2呢？？    还有关于 代码中的DCN模块对应的是deform_cony,py文件中的代码吗？非常感谢！                                                                                                                                                                                                                         offset0 = self.offsetGen0(torch.cat([rgb[0], depth[0]*2], dim=1))  # concat for offset
        offset0_up = F.interpolate(offset0, size=(int(h*0.5), int(w*0.5)), mode='bilinear', align_corners=True)
        feat_D0 = self.relu(self.dcpack_0([depth[0], offset0]) )

**Comment by iGuoYanjun** [comment]
Regarding depth[0]*2: This is an **empirical hyperparameter** used for feature scaling. Its purpose is to balance the magnitude of the two modalities (RGB and Depth) so they are on a similar scale, which helps the model fuse them more effectively.

Regarding DCN: Yes
